qube/create/templates/portlet/portlet_groovy_osgi/{{ cookiecutter.project_slug }}/src/main/webapp/WEB-INF/web.xml

```diff
@@ -17,7 +17,7 @@
 
 		<init-param>
 			<param-name>UI</param-name>
-			<param-value>life.qbic.portal.portlet.{{ cookiecutter.main_class_prefix }}Portlet</param-value>
+			<param-value>life.qbic.portal.portlet.MyPortletUI</param-value>
```

Comment:
would probably need to change to {{ cookiecutter.main_class_prefix }}PortletUI when you accidentally deleted the class above